src/api/app/services/notification_service/notifier.rb

```diff
@@ -62,32 +62,27 @@ def call
         next if channel == :rss && @event.eventtype.in?(REJECTED_FOR_RSS)
 
         @event.subscriptions(channel).each do |subscription|
-          create_notification_per_subscription(subscription, channel)
+          create_notification(subscription, channel)
         end
       end
     end
 
     private
 
-    def create_notification_per_subscription(subscription, channel)
-      return unless create_notification?(subscription.subscriber, channel)
+    def create_notification(subscription, channel)
+      # conditions to pass to create the new event notification
```

Comment:
I'm not quite sure what you are trying to say with that comment..

Reply:
Just a hint for the following multiple checks: the idea is to tell the next developer "they are just a long list to prevent the only line that matters in the method, which is the last one that create the event".

I tend to be a bit verbose to provide the reasoning logic before the code. Can I leave it or should I drop it? I'm fine with both actually

Reply:
I dont think they add any more clarity...

Reply:
Dropped